README.md

```diff
@@ -83,6 +83,7 @@ vous trouverez sur cette plateform des cours et examens qui vous accompagnerons
 * [Nodeschool](https://nodeschool.io/#workshoppers) Learn Node from the command line
 * [Video2brain](https://www.video2brain.com/) Courses online, focus on programming, animation, design, marketing and enterprise software
 * [Platzi](https://platzi.com/) Learn technology with live classes and real-time interaction
+*[Gitignore.io](https://www.gitignore.io) Create Useful .gitignore Files For Your Projects
```

Comment:
As per Markdown syntax [reference](https://help.github.com/articles/basic-writing-and-formatting-syntax/#lists), you must leave a space `" "` after the list annotator. So your code should be like
```markdown
* [Gitignore.io](https://www.gitignore.io) Create Useful 
 ^____ space goes here